poolqc/pooling_qc_sample.py

```diff
@@ -56,34 +54,33 @@
         )),
     html.Div(id='Title', style={'fontSize': 25, 'textAlign': 'center', 'padding': 30}),
     html.Div(children=''),
+    html.A(
+        'Download Data',
+        id='download-link',
+        download='rawdata.csv',
+        href='',
+        target='_blank'
+    ),
     html.Div(
         dt.DataTable(
             id='Summary Table',
             editable=True,
             row_selectable='multi',
             selected_rows=[],
             style_cell={
-                'minWidth': '150px',
-                'maxWidth': '150px',
                 'textAlign': 'center'
             },
-            n_fixed_rows=True,
             style_table={
                 'maxHeight': '1000px',
-                'overflowY': 'scroll'
+                'maxWidth': '100%',
+                'overflowY': 'scroll',
+                'overflowX': 'scroll'
```

Comment:
Please freeze the first row of the table (headers).

Reply:
Currently not possible. Jira ticket created for future implementation.
https://jira.oicr.on.ca/browse/GR-778